std-bits/table/src/main/java/org/enso/table/formatting/DateFormatter.java

```diff
@@ -3,6 +3,7 @@
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 import java.util.Locale;
+import org.graalvm.polyglot.Value;
```

Comment:
Value changes here are not needed as already encapsulated by the Column.

Reply:
alternatively the `canFormat` method should also allow Value which isDate.

Reply:
Without this change there is a test failure:
```
DataFormatter.format:
    - should handle Nothing
    - should format numbers
    - should allow customizing the decimal point and thousand separator
    - should format booleans
    - should allow custom boolean formats
    - should format dates
    - [FAILED] should allow custom date formats
        Reason: '2022-06-21' did not equal 'Tue, 21 Jun 2022' (at /home/devel/NetBeansProjects/enso/enso/test/Table_Tests/src/Data_Formatter_Spec.enso:152:13-84).
```